const error = v => { 
  throw new Error(v) 
}

export default error